spk-config.yaml

```diff
@@ -1,8 +1,9 @@
 # Azure DevOps Configuration
 azure_devops:
-  access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
+  access_token: "<my_token>" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
   hld_repository: "dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
   manifest_repository: "dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
+  infra_repository: "dev.azure.com/naros32/bedrock_testing_phase2/_git/spk-testing-private-azdo" # Repository URL that contains your terraform templates to be sed for scaffolding and generating infrastructure deployment templates.
   org: "epicstuff" # Your AzDo Org
```

Comment:
Will there ever be a situation where more than one PATs are used? (e.g. different organization, different repos)

Reply:
There may be, but we can recommend users to consolidate their templates into one private repo for the first iteration. What do you think?